hack/e2e-suite/services.sh

```diff
@@ -79,6 +79,22 @@ function make_namespace() {
 __EOF__
 }
 
+wait_for_apiserver() {
+  echo "Waiting for apiserver to be up"
+
+  local i
+  for i in $(seq 1 12); do
+    results=($(ssh-to-node "${master}" "
+      wget -q -T 1 -O - http://localhost:8080/healthz || true
```

Comment:
Ah, you're right, it's already mixed even on the ssh case.

(I was ignoring the busybox container case - there you definitely have to be careful.)